test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/SwaggerUITests.cs

```diff
@@ -1,10 +1,14 @@
+using System;
 using System.Collections.Generic;
 using System.Reflection;
-
+using System.Threading.Tasks;
 using FluentAssertions;
```

Comment:
Can we add a blank line between `using System.Threading.Tasks;` and `using FluentAssertions;`?
`